### DISCUSSION TITLE
Problem with NPT ensemble 64 water training set with PBC

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1203] [Q&A] [answered]
Hello everyone,

I'm trying to train a dataset 64 water NPT for the 5000-time frame but when I run it with DeepMD it crashed, and when I remove PBC, it runs perfectly. I used the input.json same as in the se_e3 example on the DeepMD website and the latest version of DeepMD 2.0.2.
What is the problem? also, I attached the error file.

Thanks
Saeed
[job11861122.log](https://github.com/deepmodeling/deepmd-kit/files/7326402/job11861122.log)

**Comment by saeedshahi** [comment, marked as answer]
I found out the NPT dataset does not work with Descriptor "se_e3". I can train my dataset with Descriptor "se_e2_a" easily.

**Comment by wanghan-iapcm** [comment]
Could you please provide way of reproducing the bug? thanks!

**Reply by saeedshahi** [reply]
I tried to send the NPT dataset that I got this error by email but it didn't deliver. I don't know why. so if you know another way for sending the data (about 40 Mb) please let me know.

Best,
Saeed

**Reply by wanghan-iapcm** [reply]
According to our experience the `se_3` descriptor works fine with data of changing simulation box. 

One may half the dataset and finally find the minimal dataset that can reproduce the bug. 

Your error log is mixing with the error message from `dp freeze`. Could you please clean it and only show us the error from the `dp train`? Thanks!

**Comment by saeedshahi** [comment]
Sure.
[train.log](https://github.com/deepmodeling/deepmd-kit/files/7456959/train.log)

**Reply by njzjz** [reply]
It seems to be a known issue #1062. Please try CUDA 10.1 instead.
![image](https://user-images.githubusercontent.com/9496702/139792277-80b4b1ff-222f-4761-8e69-5e47342c7076.png)
